namespaces/live-1.cloud-platform.service.justice.gov.uk/probation-tiering-dev/resources/variables.tf

```diff
@@ -0,0 +1,31 @@
+variable "application" {
```

Comment:
@nickmcmahon01 , can you please update the variables to have snake case. We are trying to keep the consistency. You can refer the CLI template file in here: https://github.com/ministryofjustice/cloud-platform-environments/blob/main/namespace-resources-cli-template/resources/variables.tf

Reply:
no problem, I copied an existing one so i'll go through and do that now.